test/log.spec.js

```diff
@@ -798,34 +790,34 @@ apis.forEach((IPFS) => {
         await log4.append('helloD5')
         await log1.append('helloA5')
         log4.join(log1)
-        assert.deepEqual(log4.clock.id, 'D')
-        assert.deepEqual(log4.clock.time, 7)
+        assert.deepStrictEqual(log4.clock.id, 'D')
+        assert.deepStrictEqual(log4.clock.time, 7)
 
         await log4.append('helloD6')
-        assert.deepEqual(log4.clock.time, 8)
-
-        const expectedData = [ 
-          { payload: 'helloA1', id: 'X', clock: { id: 'A', time: 1} },
-          { payload: 'helloB1', id: 'X', clock: { id: 'B', time: 1} },
-          { payload: 'helloD1', id: 'X', clock: { id: 'D', time: 1} },
-          { payload: 'helloA2', id: 'X', clock: { id: 'A', time: 2} },
-          { payload: 'helloB2', id: 'X', clock: { id: 'B', time: 2} },
-          { payload: 'helloD2', id: 'X', clock: { id: 'D', time: 2} },
-          { payload: 'helloC1', id: 'X', clock: { id: 'C', time: 3} },
-          { payload: 'helloC2', id: 'X', clock: { id: 'C', time: 4} },
-          { payload: 'helloD3', id: 'X', clock: { id: 'D', time: 5} },
-          { payload: 'helloD4', id: 'X', clock: { id: 'D', time: 6} },
-          { payload: 'helloA5', id: 'X', clock: { id: 'A', time: 7} },
-          { payload: 'helloD5', id: 'X', clock: { id: 'D', time: 7} },
-          { payload: 'helloD6', id: 'X', clock: { id: 'D', time: 8} },
+        assert.deepStrictEqual(log4.clock.time, 8)
+
+        const expectedData = [
+          { payload: 'helloA1', id: 'X', clock: new Clock('A', 1) },
+          { payload: 'helloB1', id: 'X', clock: new Clock('B', 1) },
+          { payload: 'helloD1', id: 'X', clock: new Clock('D', 1) },
+          { payload: 'helloA2', id: 'X', clock: new Clock('A', 2) },
+          { payload: 'helloB2', id: 'X', clock: new Clock('B', 2) },
+          { payload: 'helloD2', id: 'X', clock: new Clock('D', 2) },
+          { payload: 'helloC1', id: 'X', clock: new Clock('C', 3) },
+          { payload: 'helloC2', id: 'X', clock: new Clock('C', 4) },
+          { payload: 'helloD3', id: 'X', clock: new Clock('D', 5) },
+          { payload: 'helloD4', id: 'X', clock: new Clock('D', 6) },
+          { payload: 'helloA5', id: 'X', clock: new Clock('A', 7) },
+          { payload: 'helloD5', id: 'X', clock: new Clock('D', 7) },
+          { payload: 'helloD6', id: 'X', clock: new Clock('D', 8) }
```

Comment:
Let's keep the clock's data structure as it was (as per the reasoning in the previous comment).

Reply:
The problem with that is that it won't pass the test since the clock object has more than just `id` and a `time` properties. When it's compared to the entry, the assertion fails, is it ok if we keep it like that or should I add the missing properties here?